src/components/form/ImportModelTemplateForm.vue

```diff
@@ -87,6 +89,35 @@ async function initAllModels() {
     .filter((model) => model.plugin === props.template.plugin)
     .map(({ path }) => path);
 }
+
+/**
+ * Add template config to global config.
+ * @param {string} realModelName - Model name from plugin.
```

Comment:
why realModelName and not just modelName ? or baseModelName or globalModelName

Reply:
because `modelName` is a ref which correspond to the content of the field, and could contain `/` or `_` or other...